MediaRecorder.bs

```diff
@@ -510,18 +520,39 @@ interface MediaRecorder : EventTarget {
   <dt><dfn method for="MediaRecorder"><code>isTypeSupported(DOMString type)
     </code></dfn></dt>
   <dd>
+  This method is deprecated; it exists for legacy compatibility reasons only.
+  The <dfn>list of synchronously exposed codecs</dfn> is: <code>vp8</code>,
+  <code>vp9</code>, <code>h264</code>, and <code>opus</code>.
+  <div class="note" highlight="javascript">
+    Users are expected to discover newer codecs through {{MediaCapabilities}} instead.
+    For example:
+    <pre>
+    await navigator.mediaCapabilities.encodingInfo({
+      type: "record",
+      video: {
+        contentType: "video/webm;codecs=av01",
+        width: 1920,
+        height: 1080,
+        bitrate: 120000,
+        framerate: 48,
+      }
+    });
+    </pre>
+  </div>
```

Comment:
Should we add a note saying that due to the difficulty of detecting hardware support synchronously, the precise answer may be timing dependent?

Reply:
In practice does any browser ship one of these codecs with MediaRecorder without a sw fallback?